### PR TITLE
Ft get all requests admin api 157734038

### PR DESCRIPTION
**Pull request by LuffyKing**
#### What does this PR do?
Adds a getAllRequests API for admin users registered on the app to allow them to view all repair or maintenance requests,  and robust tests for said API.

#### Description of Task to be completed?
Addition of robust tests to ensure edge cases are accounted for and handled properly.  getAllRequestsAdmin API created to allow admin users, who are registered on the app, to view all the requests on the platform. This feature makes it easy to see which requests need to be attended to.
#### Any background context you want to provide?
Robust testing is the key to a solid application.
#### What are the relevant Github Issues?
The only issue so is only the pending review
Questions:

### CODE REVIEW

File: server/controllers/Requests.js [hunk 1]
```diff
@@ -35,6 +35,34 @@ const Requests = {
     });
   },
   /**
+* It gets all the requests on the application for an admin user
+* @param {Object} request - request object containing params and body
+* @param {Object} response - response object that conveys the result of the request
+* @returns {Object} - response object that has a status code of 200 as long as a
```
Comment (Tawakalt) [inline server/controllers/Requests.js:41]: You omitted the name of the returned object before the description.

File: server/controllers/Requests.js [hunk 2]
```diff
@@ -35,6 +35,34 @@ const Requests = {
     });
   },
   /**
+* It gets all the requests on the application for an admin user
```
Comment (Tawakalt) [inline server/controllers/Requests.js:38]: There is a Jsdoc property for `description` you know.
